extensions/common/context.h

```diff
@@ -174,12 +182,12 @@ struct RequestInfo {
   std::string url_scheme;
 
   // TCP variables.
-  int64_t tcp_connections_opened = 0;
-  int64_t tcp_connections_closed = 0;
-  int64_t tcp_sent_bytes = 0;
-  int64_t tcp_received_bytes = 0;
-  int64_t tcp_total_sent_bytes = 0;
-  int64_t tcp_total_received_bytes = 0;
+  uint8_t tcp_connections_opened = 0;
```

Comment:
in case of a leak, this can go beyond 255.. -> https://github.com/istio/istio/issues/24720#issuecomment-674147639